src/js/vic-v2/components/IntroductionPage.jsx

```diff
@@ -10,63 +10,143 @@ class IntroductionPage extends React.Component {
   componentDidMount() {
     focusElement('.va-nav-breadcrumbs-list');
   }
+
   goForward = () => {
     this.props.router.push(this.props.route.pageList[1].path);
   }
+
+  renderSignInMessage = () => {
+    return (
+      <div>
+        <div className="usa-alert usa-alert-info">
+          <div className="usa-alert-body">
+            If you’re signed in to your account, the application process can go more smoothly. Here’s why:
+            <br/>
+            <ul>
+              <li>We can prefill part of your application based on your account details.</li>
+              <li>You can save your form in progress, and come back later to finish filling it out.</li>
+              <li>You could get your card sooner because we can confirm your Veteran status more quickly.</li>
+            </ul>
+            <p><button className="va-button-link" onClick={() => this.props.saveInProgressActions.toggleLoginModal(true)}>Sign in to your account.</button></p>
+          </div>
+        </div>
+        <br/>
+      </div>
+    );
+  }
+
   render() {
+    // TODO: Hook this up to the right user check
+    const idProofed = false;
+    const signedIn = this.props.saveInProgress.user.login.currentlyLoggedIn;
+    const accountBenefits = (
+      <ul>
+        <li>We can prefill part of your application based on your account details.</li>
+        <li>You can save your form in progress, and come back later to finish filling it out.</li>
+        <li>You can get your card sooner because we can confirm your Veteran status more quickly.</li>
+      </ul>
+    );
+    const idProofingReqs = (
+      <div>
+        <div>To go through the ID.me identity-proofing process, you’ll need:</div>
+        <ul>
+          <li>A smartphone (or a landline or mobile phone and a computer with an Internet connection)</li>
+          <li>Your Social Security number</li>
+          <li>Proof of your identity (which could be a driver’s license, passport, or the ability to answer questions based on private and public data (like your credit report))</li>
+        </ul>
+        <p>Verifying your identity is a one-time process that will take about 5-10 minutes. Once you’ve gone through the identity process, you won’t need to do it again. Signing in next time will be much faster.</p>
```

Comment:
@peggygannon, do you think it might make sense to join the last two sentences? Once you’ve gone through the identity process, you won’t need to do it again and signing in next time will be much faster.

Reply:
@cehsu Please leave as 2 sentences. This structure adheres to our plain-language guidelines. In addition, this copy mirrors copy that's currently on our log in page, so we want to re-affirm the messaging.